### DISCUSSION TITLE
Seo releted

### DISCUSSION THREAD

**Discussion by kiranwebiots** [tailwindlabs/tailwindcss#7472] [Help]
tailwind is better or bootstrap

 I'm asking tailwind about performance. Tailwind making everything easier for my idea. Also, your script is so fast. It's fast but not SEO-friendly. Can you suggest something to make it more SEO-friendly?

**Comment by dotnetCarpenter** [comment]
CSS usually has no impact on Search Engine Optimization (SEO). Could you be more specific in your question?

The usual installation of Tailwind CSS is to compile your CSS on your machine before uploading it to a web server. The speed of compiling can not have any impact on user experience, SEO or your web server.

Here is a guide on writing SEO HTML: https://html.com/semantic-markup/
As you can read, CSS has zero impact on SEO.

> tailwind is better or bootstrap

In regard to SEO, there is no difference.
